src/components/Employee/Compensation/Compensation.tsx

```diff
@@ -31,9 +36,13 @@ import {
 } from '@/api/queries'
 import { RequireAtLeastOne } from '@/types/Helpers'
 
-export type CompensationDefaultValues = RequireAtLeastOne<
-  Partial<Pick<Schemas['Job'], 'rate' | 'title' | 'payment_unit'>>
->
+export type CompensationDefaultValues = RequireAtLeastOne<{
+  rate?: Schemas['Job']['rate']
+  title?: Schemas['Job']['title']
+  payment_unit?: (typeof PAY_PERIODS)[keyof typeof PAY_PERIODS]
+  flsa_status?: Schemas['Compensation']['flsa_status']
+}>
```

Comment:
Picking compensation values how we were doing previously was actually not correctly limiting the default values. Ex see

<img width="1235" alt="Screenshot 2025-03-14 at 4 39 16 PM" src="https://github.com/user-attachments/assets/1cf75254-ac86-4780-b655-b3279be46682" />

Not sure the best approach beyond picking, but this intentionally enumerates what's available instead